static/.asf.yaml

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
We should keep this file in `camel-website-pub` repository

Reply:
On this I disagree with you.  If we keep it here, publishing consists of removing _everything_ from the site-pub repo, adding the entire site, and committing.  If we keep it in  the `camel-website-pub` repo then we have to make an exception for one file.  I think that is excessively complex for no gain.

Reply:
As far as I understand the [documentation](https://infra.apache.org/project-site.html) on publishing the website, the built website from camel-website-pub repository will not be published without it. This is a requirement for publishing the website that the git repository with the website content contains `.asf.yaml` file.

Reply:
If we keep it in static/.asf.yaml, it will be placed in the correct branch in camel-website-pub with every commit. I set up the Aries and Felix websites this way and they seem to work fine.

Reply:
Oh, that's actually quite clever, I missed that, apologies :+1: